modules/vsphere-problem-detector-running.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_vsphere/using-vsphere-problem-detector-operator.adoc
+
+:operator-name: vSphere Problem Detector Operator
+
+[id="vsphere-problem-detector-running_{context}"]
+= Running the vSphere Problem Detector Operator checks
+
+The {operator-name} automatically runs the checks every 8 hours. However, when the Operator starts, it runs the checks immediately. The Operator is started by the Cluster Storage Operator when the Cluster Storage Operator starts and determines that the cluster is running on vSphere. To run the checks immediately, you can scale the {operator-name} to `0` and back to `1` so that it restarts the {operator-name}.
+
+.Procedure
+
+. Scale the Operator to `0`.
++
+[source,terminal]
+----
+$ oc scale deployment/vsphere-problem-detector-operator --replicas=0 \
+    -n openshift-cluster-storage-operator
+----
++
+If the deployment does not scale to zero immediately, you can run the following command to wait for the pods to exit.
++
+[source,terminal]
+----
+$ oc wait pods -l name=vsphere-problem-detector-operator \
+    --for=delete --timeout=5m -n openshift-cluster-storage-operator
+----
+
+. Scale the Operator back to `1`.
++
+[source,terminal]
+----
+$ oc scale deployment/vsphere-problem-detector-operator --replicas=1 \
+    -n openshift-cluster-storage-operator
+----
+
+. Delete the old leader lock to speed up the new leader election for the Cluster Storage Operator.
++
+[source,terminal]
+----
+$ oc delete -n openshift-cluster-storage-operator \
+    cm vsphere-problem-detector-lock
+----
+
+.Verification steps
+
+* View the events or logs that are generated by the {operator-name}.
```

Comment:
What is the use looking for in the logs?

Reply:
In this context of running the checks manually, I think the verification step can be to look for recent timestamps on either the events or the logs.  Instead of showing sample logs here, I'll share that in the module that covers viewing the logs.